package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Order_Tracking",
+  "version": "1.0.0",
```

Comment:
Consider adding a more descriptive name for your application. This will help other developers understand the purpose of your application at a glance.